app/frontend/src/components/FileInput.vue

```diff
@@ -17,21 +17,23 @@
               <v-card-text>
                 <v-tabs-items v-model="templateTab">
                   <v-tab-item>
-                    <p>Upload your template file.</p>
                     <v-file-input
                       counter
                       :clearable="true"
                       label="Upload template file"
+                      hint="See below for sample templates and supported formats"
+                      persistent-hint
                       prepend-icon="attachment"
                       required
-                      :rules="notEmpty"
+                      mandatory
+                      :rules="(this.templateTab === 1) ? [] : notEmpty"
                       show-size
                       v-model="form.files"
                     />
                   </v-tab-item>
                   <v-tab-item md6>
```

Comment:
This md6 is intended? Looks like it may be a copy-paste typo.

Reply:
good call!